public/js/RoomClient.js

```diff
@@ -2531,9 +2530,6 @@ class RoomClient {
                 pn = document.createElement('button');
                 pn.id = id + '__pin';
                 pn.className = html.pin;
-                ha = document.createElement('button');
```

Comment:
hideAll button - line 2534 RoomClient.js

Reply:
likely old code.  I don't recall needing it for anything.